buildkite/bazelci.py

```diff
@@ -2290,6 +2295,26 @@ def set_env_var(config_key, env_var_name):
     print_pipeline_steps(pipeline_steps, handle_emergencies=not is_downstream_project)
 
 
+def show_gerrit_review_link(git_repository, pipeline_steps):
+    host = git_repository[len("https://"): git_repository.find(".googlesource")]
+    text = "The transformed code used in this pipeline can be found under https://{}-review.googlesource.com/q/{}". \
+        format(host, os.getenv("BUILDKITE_COMMIT"))
+    commands = ["buildkite-agent annotate --style=info '{}'".format(text)]
+    pipeline_steps.append(
+        create_step(
+            label=":pipeline: Print information about Gerrit Review Link",
+            commands=commands,
+            platform=DEFAULT_PLATFORM,
+        )
+    )
+
+
+def is_git_on_borg_repo(git_repository):
+    if git_repository is not None and "googlesource.com" in git_repository:
```

Comment:
This can be shortened:

return git_respository and "googlesource.com" in git_repository